_posts/2019-12-30-3-dep-lidar-proposal-2020.md

```diff
@@ -0,0 +1,35 @@
+---
+title: '3DEP Lidar Proposal for 2020'
+author:
+  display_name: Rick Kelson
+  email: rkelson@utah.gov
+date: 2019-10-29 22:15:55
+categories:
+- Featured
+tags:
+- lidar
+- 3dep
+---
+
+In October a partnership of Utah organizations submitted a proposal to the [USGS 3D Elevation Program (3DEP) FY19/20 Broad Agency Announcement (BAA)](https://www.usgs.gov/core-science-systems/ngp/3dep/fy1920-usgs-broad-agency-announcement-baa-3d-elevation-program-3dep) for over 18,217 square miles for 2020 Lidar acquisition projects. A couple projects, Kane County (Greater Sevier Addition) and Strawberry River, have already begun.
+
+While we're months away from any preliminary or final funding decisions, the partnership is sharing [this proposed project area map]({{ "/images/3DEP_Lidar_Proposal_2020.png" | prepend: site.baseurl }})
+ in advance, in an effort to promote coordination and avoid duplication of effort in these areas.
+
+The proposed areas break down to the following square mileage:
+[![3DEP Proposal]({{"/images/3DEP_Lidar_Proposal_2020_sm.png" | prepend:site.baseurl}} "2020 3DEP Lidar Proposal A - click for larger image")]({{"/images/3DEP_Lidar_Proposal_2020.png" | prepend:site.baseurl}}){:.inline-text-right}
+
+Projects Already Underway
+
+- Kane County (QL2) - 6,055
+- Strawberry River (QL1 & 4-band aerial photography) - 88
+
+Proposed Projects
+
+- Wasatch Back (QL1 or QL2) - 1,103
+- Iron County (QL1 & QL2) - 3,067
+- NRCS (QL1 or QL2) - 2,706
+- Sanpete & Sevier County (QL2) - 2,397
```

Comment:
Should this be Sanpete and Sevier Counties?

Reply:
probably

```suggestion
- Sanpete and Sevier Counties (QL2) - 2,397
```